src/transform.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+const babel = require('@babel/core');
+const fs = require('fs');
+
+// addSetupNode adds the setup() call to the beginning of the script.
+function addSetupNode() {
+  const setupNode = babel.template.statement(`
+    setup({single_test: true, explicit_done: false});
+  `)();
```

Comment:
Maybe you've already known this; IIUC, this is the same as
```suggestion
  const setupNode = babel.template.statement.ast(`
    setup({single_test: true, explicit_done: false});
  `);
```

(Personally I don't have a strong preference between the two.)